reqs/tests/link_agencies_tests.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from model_mommy import mommy
+
+from reqs.management.commands import link_agencies
+from reqs.management.commands.sync_agencies import Command as SynchCommand
```

Comment:
I blame you for this, since you named it `sync_agencies`. (I'll probably get around this by not providing an alias, since refactoring I did after initially importing it like that means it's not really necessary.)